README.md

```diff
@@ -121,6 +121,23 @@ If a secret is provided, then it is assumed that the connection to Git requires
 
 If the `uri` is not specified in the `parameterSource` section, then it will default to the `uri` specified under `templateSource`.
 
+### parameterSource processing
+Eunomia uses the [yq command](http://mikefarah.github.io/yq/) to merge all yaml files in the specified folder. You have to be careful, if you have the same variable name in multiple files. Dictionaries will merge, lists will get overwritten. 
+
+#### Variable Hierarchy
+You can provide a file `hiera.lst`, to allow a variable hierarchy. This will allow for example to specify a default value and overwrite it on an environment level if necessary. This will greatly simplify your configuration and allows for deduplication of data, making your operational life a lot easier.
```

Comment:
Why `hiera` and not `hierarchy`? is it some kind of standard name? It seems to me that in the project we  tend to keep full names everywhere with surprising consistency (at least I don't recall many abbreviations), which has the nice benefit of making remembering how to write stuff a bit easier.